src/routes/home/index.js

```diff
@@ -2,5 +2,7 @@ const HomeView = require('./components/HomeView');
 
 // Sync route definition
 module.exports = {
-    component: HomeView
+    path: '/',
+    component: HomeView,
+    exact: true
```

Comment:
`exact: true` is key to render the root route. If this isn't set, the HomeView will render on both paths: `/` and `/counter`

Reply:
For 404 page functionality, I'm wondering if we set it to `/` with exact set to false